doc/installation.rst

```diff
@@ -187,7 +187,7 @@ We recommend you install the above packages via your system's software managemen
 EOS requires the following scientific software:
 
 GSL
-  the GNU Scientific Library \cite{GSL}, in version 1.16 or higher;
+  the GNU Scientific Library \cite{GSL}, in version 1.15 or higher;
```

Comment:
Why do we include an earlier version here?

Reply:
Good catch. Because it actually builds with version 1.15 and that is what we currently use at IPPP.

Reply:
If it works for 1.15, then this change is justified